src/style.css

```
@@ -1,3 +1,147 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00acdc;
+  --second-color: black;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  background-color: #f4f4f4;
+  width: 100%;
+  font-weight: 500;
+  height: auto;
+}
+
+header {
```

Use only class selector everywhere 
```suggestion
.header {
```